### ISSUE TITLE
Research useful diagnostics for emulators beyond simple metrics / scatterplots

### ISSUE THREAD

**Issue by kallewesterling** [alan-turing-institute/autoemulate#72]

**Comment by mastoffel** [comment]
> @mastoffel do you think this might be a good idea for @bryanlimy to start looking at?

I think for @bryanlimy there are a few things to look at on the `PyTorch` side. I just added an issue #128 and there'll be something else too.

**Comment by mastoffel** [comment]
Also, just to add, I've talked to Ieva Kazlauskaite from BAS, who's an expert in emulation and would be curious about discussing diagnostics, so I should contact her about this soon.

**Comment by kallewesterling** [comment]
> I think for @bryanlimy there are a few things to look at on the PyTorch side. I just added an issue https://github.com/alan-turing-institute/autoemulate/issues/128 and there'll be something else too.

Great, thanks! :)

> Also, just to add, I've talked to Ieva Kazlauskaite from BAS, who's an expert in emulation and would be curious about discussing diagnostics, so I should contact her about this soon.

This sounds like an exciting development as well. Can I help somehow?

**Comment by mastoffel** [comment]
I'm wondering whether this could be part the literature review @aranas ? How do we evaluate emulators apart from simply looking at plots and r2/rsme values? Or are they sufficient?

**Comment by aranas** [comment]
yes agreed, this could be informed by the lit review for benchmarks.
also to document here that from past conversations this particular question came up as a point of interest for @zackxconti, re evaluation metrics beyond r2. And specific to sensitivity analysis pipeline for parameter estimation in cardiac modelling @MaxBalmus and I talked about how often the rank order of parameters and the stability thereof may be a relevant criteria.

**Comment by mastoffel** [comment]
yes, so things like that would be very good to know!
